core/src/main/java/io/grpc/transport/ClientTransport.java

```diff
@@ -63,7 +63,7 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
                          ClientStreamListener listener);
 
   /**
-   * Starts transport. Implementations must not call {@code listener} until after {@code start()}
+   * Starts transport. Implementations must not call {@code listener} until after {@link #start}
```

Comment:
it does add a little value, in case the method name ever changes.  Also, cross references look at the @link annotation (such as in CS and IDEs).